python/lsst/sconsUtils/tests.py

```diff
@@ -328,12 +336,15 @@ def runPythonTests(self, pyList):
                     continue
                 pythonTestFiles.append(os.path.join(self._cwd, f))
 
+        # If user has set an envvar to control warnings, defer to them.
+        warn_opt = "-Wd " if "PYTHONWARNINGS" not in os.environ else ""
+
         # Now set up the python testing target
         # We always want to run this with the tests target.
         # We have decided to use pytest caching so that on reruns we only
         # run failed tests.
         lfnfOpt = "none" if 'install' in SCons.Script.COMMAND_LINE_TARGETS else "all"
-        interpreter = f"pytest -Wd --lf --lfnf={lfnfOpt}"
+        interpreter = f"pytest {warn_opt}--lf --lfnf={lfnfOpt}"
```

Comment:
I don't know why we have two versions of this line, but I'd rather not try to remove the duplication on this ticket.

Reply:
one is for running the standalone tests and the other is for the "run them all together" tests.